infrastructure/pre_req_delete.sh

```diff
@@ -69,7 +69,48 @@ echo "Creator Email: $creator_email"
 # shellcheck disable=SC2154
 echo "Namespace: $namespace"
 
+# Optionally, delete all versions in the S3 bucket
```

Comment:
Do you need deletion operation for s3 bucket @samanXgrid @muhdsaad @faheem-khau9? I think it is unsafe option.

Reply:
In hindsight, I'm inclined to agree. 

I just had to do it so many times, that I thought of including it. 
But this might not be the place...

Reply:
Removed the commit from the branch.